### PR TITLE
Archive GPU Feature Discovery

### PR DESCRIPTION
**Pull request by ArangoGutierrez**
Device Plugin v0.15.0 will be the first release with the new features of GPU Feature discovery, marking this repository archived

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,59 +1,3 @@
-# Contribute to the NVIDIA GPU Feature Discovery Project
+## Contributions
```
Comment (elezar) [inline CONTRIBUTING.md]: Are we allowed to just remove `CONTRIBUTING.md` like this. Is the DCO notice not still required?

**Review by elezar — COMMENTED** [review]
One question:

What is our strategy if a `v0.8.x` patch release is required?

File: CONTRIBUTING.md (hunk 1 above)
Comment (ArangoGutierrez) [reply CONTRIBUTING.md]: DCO is for when accepting contributions, is not a license per se. I am not removing the LICENSE file. and I even mention the License at the edited README

**Review by elezar — COMMENTED** [review]
One thing to consider. It would be good to link directly to the instructions to install GFD in the device plugn repo. Is that already available (I believe it might have been commented).

**Comment by ArangoGutierrez** [comment]
> One thing to consider. It would be good to link directly to the instructions to install GFD in the device plugn repo. Is that already available (I believe it might have been commented).

https://github.com/NVIDIA/k8s-device-plugin/pull/489

**Comment by ArangoGutierrez** [comment]
Created https://github.com/NVIDIA/k8s-device-plugin/pull/566

**Comment by ArangoGutierrez** [comment]
Now that https://github.com/NVIDIA/k8s-device-plugin/pull/566 is in and `k8s-device-plugin` has released `v0.15` is time to refresh this PR

**Comment by ArangoGutierrez** [comment]
Ping @elezar

File: README.md [hunk 2]
```diff
@@ -1,454 +1,35 @@
...
-    outputFile: /etc/kubernetes/node-feature-discovery/features.d/gfd
-    sleepInterval: 60s
-```
-
-The fields in this section map to the command line flags and environment
-variables available for configuring GFD.
-
-For details on how to deploy GFD as part of the plugin please refer to:
-* https://github.com/NVIDIA/k8s-device-plugin/tree/v0.12.0/#deploying-with-gpu-feature-discovery-for-automatic-node-labels
+This repository contains the source code and documentation for GPU Feature Discovery. As of [26 Feb 2024](https://github.com/NVIDIA/k8s-device-plugin/releases/tag/v0.15.0-rc.1), this project has been **archived**. Development will now continue at [k8s-device-plugin](https://github.com/NVIDIA/k8s-device-plugin).
```
Comment (elezar) [inline README.md]: Is this date correct?

```suggestion
This repository contains the source code and documentation for GPU Feature Discovery. As of [26 Feb 2024](https://github.com/NVIDIA/k8s-device-plugin/releases/tag/v0.15.0), this project has been **archived**. Development will now continue at [k8s-device-plugin](https://github.com/NVIDIA/k8s-device-plugin).
```

File: README.md [hunk 3]
```diff
@@ -1,454 +1,35 @@
...
-- [Deployment via `helm`](#deployment-via-helm)
-    + [Installing via `helm install`](#installing-via-helm-install)
-      - [Deploying Standalone](#deploying-standalone)
-      - [Deploying as a subchart of the NVIDIA device plugin](#deploying-as-a-subchart-of-the-nvidia-device-plugin)
-    + [Deploying via `helm install` with a direct URL to the `helm` package](#deploying-via-helm-install-with-a-direct-url-to-the-helm-package)
-- [Building and running locally with Docker](#building-and-running-locally-with-docker)
-- [Building and running locally on your native machine](#building-and-running-locally-on-your-native-machine)
+# GPU Feature Discovery
 
 ## Overview
```
Comment (elezar) [inline README.md]: Do we want to add a better heading such as:
```
**MIGRATION Notice**
```

**Review by elezar — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -1,454 +1,35 @@
...
-    + [Installing via `helm install`](#installing-via-helm-install)
-      - [Deploying Standalone](#deploying-standalone)
-      - [Deploying as a subchart of the NVIDIA device plugin](#deploying-as-a-subchart-of-the-nvidia-device-plugin)
-    + [Deploying via `helm install` with a direct URL to the `helm` package](#deploying-via-helm-install-with-a-direct-url-to-the-helm-package)
-- [Building and running locally with Docker](#building-and-running-locally-with-docker)
-- [Building and running locally on your native machine](#building-and-running-locally-on-your-native-machine)
+## Archived Status
 
-## Overview
+This project is no longer actively maintained. While the codebase remains available for reference purposes, no further updates or contributions are expected.
```
Comment (elezar) [inline README.md]: This is not true. GFD as a project is still maintained, it's just not developed in this repo.

File: README.md [hunk 5]
```diff
@@ -1,454 +1,35 @@
...
-The first step is to make sure that [Node Feature Discovery](https://github.com/kubernetes-sigs/node-feature-discovery)
-is running on every node you want to label. NVIDIA GPU Feature Discovery use
-the `local` source so be sure to mount volumes. See
-https://github.com/kubernetes-sigs/node-feature-discovery for more details.
+Any issues or feature requests should be submitted to the issue tracker of the new repository. 
 
-You also need to configure the `Node Feature Discovery` to only expose vendor
-IDs in the PCI source. To do so, please refer to the Node Feature Discovery
-documentation.
+## License
```
Comment (elezar) [inline README.md]: Is it required to add a license section?

File: README.md
Comment (elezar) [inline README.md:1]: How about just updating this file to:
```
# MIGRATION NOTICE

With the [v0.15.0](https://github.com/NVIDIA/k8s-device-plugin/releases/tag/v0.15.0)
release of the NVIDIA GPU Device Plugin and GPU Feature Discovery, the migration
of development for GPU Feature Discovery to the
https://github.com/NVIDIA/k8s-device-plugin is complete.

This repository is archived and is only maintained for reference purposes.

## Issues and Contributing

[Checkout the Contributing document!](https://github.com/NVIDIA/k8s-device-plugin/blob/main/CONTRIBUTING.md)

For questions, feature requests, or bugs, open an issue against the [k8s-device-plugin](https://github.com/NVIDIA/k8s-device-plugin) repository.

```

Comment (ArangoGutierrez) [reply README.md:1]: Thanks for the recommendation, added

File: README.md (hunk 5 above)
Comment (ArangoGutierrez) [reply README.md]: rewritten

File: README.md (hunk 4 above)
Comment (ArangoGutierrez) [reply README.md]: rewritten

**Comment by ArangoGutierrez** [comment]
@elezar / @klueska  PTAL

File: CONTRIBUTING.md (hunk 1 above)
Comment (elezar) [reply CONTRIBUTING.md]: Let's just keep this file as is. Archiving disables contributions anyway, and I would rather not remove this.

Comment (ArangoGutierrez) [reply CONTRIBUTING.md]: ok

Comment (ArangoGutierrez) [reply CONTRIBUTING.md]: done

**Review by elezar — APPROVED** [review]

**Pull request merged**